src/dll/Hooks/ExecuteProcess.cpp

```diff
@@ -92,16 +92,19 @@ bool ExecuteScc(SccApi& scc)
     auto r6Dir = App::Get()->GetPaths()->GetR6Dir();
     auto scriptSystem = App::Get()->GetScriptCompilationSystem();
 
-    auto settings = scc.settings_new(r6Dir.string().c_str());
+    auto r6DirStr = r6Dir.u8string();
+    auto settings = scc.settings_new(reinterpret_cast<const char*>(r6DirStr.c_str()));
```

Comment:
I understand why this is done, but does redscript's API handle this?

Reply:
yes @WopsS, redscript expects it to be a UTF-8 zero-terminated string

Reply:
Ah thanks.